src/prefect/server/database/interface.py

```diff
@@ -66,6 +66,18 @@ async def run_migrations_downgrade(self):
         """Run all downgrade migrations"""
         await run_sync_in_worker_thread(alembic_downgrade)
 
+    async def is_db_connectable(self):
+        """
+        Returns boolean indicating if the database is connectable.
+        This method is used to determine if the server is ready to accept requests.
+        """
+        engine = await self.engine()
+        try:
+            async with engine.connect():
```

Comment:
What kind of timeout is there on this? Is the `/ready` endpoint going to take 60s if it's not available?

Reply:
i want to say this is dependent on the underlying engine, i couldn't find any exact literature on the a default timeout value here.  during my local testing, this timed out a 5s - but assuming this is used in k8s or similar, usually those probes have their own timeouts, after which a probe is considered to be failed

Reply:
Hm okay. I know we'll wait quite a while for a SQLite connection. We may want to add an `with anyio.fail_after(...)` timeout in the future but this seems okay for now :)

Reply:
I assumed this would use the value of `PREFECT_API_DATABASE_CONNECTION_TIMEOUT`: https://github.com/PrefectHQ/prefect/blob/91105c475c5688d520b4b4de1e548989d533f581/src/prefect/settings.py#L922

Reply:
heh, that's probably it

Reply:
Sounds right lol — great!